policyengine_us/parameters/gov/states/ar/tax/income/main.yaml

```diff
@@ -0,0 +1,33 @@
+description: Arkansas taxes personal taxable income at these rates.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Arkansas income tax rate
+  reference:
+  - title: Tax Legislation for 2022
+    href: https://www.arkansashouse.org/news/post/15098/tax-legislation-for-2022#:~:text=In%20the%20most%20recent%20special,rate%20is%20reduced%20to%205.5%25.
+  - title: State of Arkansas Estimated Tax Declaration Vouchers and Instructions for Tax Year 2023
```

Comment:
Can you add a legal code reference? (Differs from tax legislation)

Might help with the special $87k-91.9k zone